src/main/bin/ccm-fetch

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/perl ## no critic ((RequireVersionVar);
```

Comment:
I am not in favour of having to modify every single file with this per-line opt-out. Why not just disable the policy globally in PerlCritic? It's only activated at level 2 (the second highest) and perlcritic tends to get annoying and less useful at level 3, so I can't see us ever wanting to enable level 2 policies.